### ISSUE TITLE
ImGui::InputText bug when clicking out it hangs up the window

### ISSUE THREAD

**Issue by PatchByte** [ocornut/imgui#5264]
When using ImGui::InputText and then selecting the InputText box and then clicking out of it, it hangs up the window!
And it's just loading and loading and no response!
```c++
if (ImGui::BeginMenu("Client"))
		{
			ImGui::InputText("##MyNickname", g_LocalChatPlayer.nickname, sizeof(g_LocalChatPlayer.nickname) - 1);
			ImGui::EndMenu();
		}
```

**Comment by PatchByte** [comment]
It also hangs up when i just try:
```c++
static char bbuf[101] = { 0 };
if(ImGui::Begin("Test"))
{
ImGui::InputText("TestInput", bbuf, 100);
}
ImGui::End();
```

**Comment by PatchByte** [comment]
Ok i just comment 2 lines out
https://github.com/ocornut/imgui/blob/709bc03d6f07467fe10f892f9198913f0298b58e/imgui.cpp#L4785-L4788

**Comment by ocornut** [comment]
Looks like IME handler is having an issue, are you sure it isn’t your debugger stalling on loading symbols for the IME DLL ?

Since you ignored issue templates and are not providing much requested information I have little patience in helping you.

**Comment by veteri** [comment]
> Ok i just comment 2 lines out. And well now it works 👍
> 
> https://github.com/ocornut/imgui/blob/709bc03d6f07467fe10f892f9198913f0298b58e/imgui.cpp#L4785-L4788

Having the same issue on 1.88 dx9 win32 and lots of others seem to as well. 
Commenting the 2 lines out fixes it.

**Comment by bidengaming** [comment]
Happens to me aswell removing that also fixed it, without debugger attached btw

**Comment by ocornut** [comment]
Pushed fix 133bbafa3cc7b7a1ca68808dacd1f6cee2552d4f
